trl/trainer/judges.py

```diff
@@ -135,6 +135,49 @@ def judge(self, prompts: List[str], completions: List[List[str]], shuffle_order:
         raise NotImplementedError("Judge subclasses must implement the `judge` method.")
 
 
+class BaseBinaryJudge(BaseJudge):
+    """
+    Base class for binary judges.
+    """
+
+    @abstractmethod
+    def judge(
+        self, prompts: List[str], completions: List[str], gold_answers: List[str] = None, shuffle_order: bool = True
```

Comment:
maybe `"gold_completion"` or even `"ref_completion"` would be more suited, what do you think?

Reply:
`"gold_completions"` seems good to me. I have done the update.